includes/base/controls-stack.php

```diff
@@ -90,6 +90,18 @@ abstract class Controls_Stack extends Base_Object {
 	 */
 	private $config;
 
+	/**
+	 * The additional configuration.
+	 *
+	 * Holds additional configuration that has been set using the `set_config`
+	 * method and later merged into the primary config.
```

Comment:
Explain why it needed to be merged later

Reply:
Done.